mmedit/models/extractors/lte.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, requires_grad=True, pixel_range=1., pretrained=None):
             pixel_range=pixel_range, img_mean=vgg_mean, img_std=vgg_std)
 
         # use vgg19 weights to initialize
-        vgg_pretrained_features = models.vgg19(pretrained=True).features
+        vgg_pretrained_features = models.vgg19(pretrained=False).features
```

Comment:
What's the motivation of this change?

Reply:
Avoid downloading VGG in unittest of CI.